feature-utils/poly-look/test/utils/history-stub.js

```diff
@@ -0,0 +1,19 @@
+export class HistoryStub {
+  constructor() {
+    this._route = null;
+    this._state = null;
```

Comment:
Is this usually created this way? Can't it be created with this set already?

Reply:
No it's created by the useState hook, which only works in react components or contexts. It's just for testing. Could be but that wouldn't change anything would it?